src/renderer/library/components/settings/KeyboardSettings.tsx

```diff
@@ -146,6 +168,69 @@ class KeyboardSettings extends React.Component<IProps, IState> {
     public render(): React.ReactElement<{}> {
         const { __ } = this.props;
 
+        const isSearchEmpty = !this.state.searchItem || this.state.searchItem.trim() === "";
+
+        const cleanNames = {
```

Comment:
you need to type the `cleanNames` object with : https://github.com/edrlab/thorium-reader/blob/00cc65a19f9e3801e2da312e569bd7e1f62e3b10/src/common/keyboard.ts#L400C13-L400C32